atom/browser/net/atom_network_delegate.cc

```diff
@@ -146,6 +146,12 @@ void ToDictionary(base::DictionaryValue* details, net::URLRequest* request) {
   details->SetString("resourceType",
                      info ? ResourceTypeToString(info->GetResourceType())
                           : "other");
+  net::IPEndPoint request_ip_endpoint;
+  bool was_successful = request->GetRemoteEndpoint(&request_ip_endpoint);
```

Comment:
just a nit, but typically with boolean return values the method call goes in the `if` statement